src/components/Template/Template.mdx

```diff
@@ -5,496 +5,123 @@ import { changelogData } from "./templateChangelogData";
-        // ...
-      </TemplateAboveHeader>
-      <TemplateHeader>
-        // ...
-      </TemplateHeader>
-    </TemplateBreakout>
-    <TemplateContent sidebar="...">
-      <TemplateContentTop>
-        // ...
-      </TemplateContentTop>
-      <TemplateContentPrimary>
-        // ...
-      </TemplateContentPrimary>
-      <TemplateContentBottom>
-        // ...
-      </TemplateContentBottom>
-    </TemplateContent>
-    <TemplateFooter>
-      // ...
-    </TemplateFooter>
+  <Template sidebar="...">
+    <TemplateBreakout>...</TemplateBreakout>
+    <TemplateTop>...</TemplateTop>
+    <TemplateSidebar>...</TemplateSidebar>
+    <TemplateMain>...</TemplateMain>
+    <TemplateBottom>...</TemplateBottom>
   </Template>
 </>
-`}
-  language="jsx"
-/>
+`} language="jsx" />
 
-## Template Children Component Props
+## Template Props
 
-<Canvas of={TemplateStories.ChildrenComponentProps} />
+<Canvas of={TemplateStories.WithControls} />
 
-<ArgTypes of={TemplateStories.ChildrenComponentProps} />
+<ArgTypes of={TemplateStories.WithControls} />
 
-### Template Components
+## Template Components
 
 The components consist of, listed in DOM structure order:
 
 - `Template`
-- `TemplateAboveHeader`
-- `TemplateHeader`
 - `TemplateBreakout`
-- `TemplateContent`
-- `TemplateContentTop`
-- `TemplateContentPrimary`
-- `TemplateContentSidebar`
-- `TemplateContentBottom`
-- `TemplateFooter`
-
-#### Template
-
-<Source
-  code={`
-<Template>
-  // ...
-</Template>
-`}
-  language="jsx"
-/>
-
-<Description of={Template} />
-
-#### TemplateBreakout
-
-<Source
-  code={`
-<Template>
-  <TemplateBreakout>
-    // ...
-  </TemplateBreakout>
-</Template>
-`}
-  language="jsx"
-/>
-
-<Description of={TemplateBreakout} />
-
-#### TemplateAboveHeader
-
-<Source
-  code={`
-<Template>
-  <TemplateBreakout>
-    <TemplateAboveHeader>
-      // ...
-    </TemplateAboveHeader>
-  </TemplateBreakout>
-</Template>
-`}
-  language="jsx"
-/>
-
-<Description of={TemplateAboveHeader} />
+- `TemplateTop`
+- `TemplateSidebar` (if meant to render on left)
+- `TemplateMain` or `TemplateMainNarrow`
+- `TemplateSidebar` (if meant to render on right)
+- `TemplateBottom`
 
-#### TemplateHeader
+### Using the `Template` with the `Header` and `Footer`
 
-<Source
-  code={`
-<Template>
-  <TemplateBreakout>
-    <TemplateAboveHeader>
-      // ...
-    </TemplateAboveHeader>
-    <TemplateHeader>
-      // ...
-    </TemplateHeader>
-  </TemplateBreakout>
-</Template>
-`}
-  language="jsx"
-/>
+TBD.
 
-<Description of={TemplateHeader} />
+## Template Accessibility
 
-#### TemplateContent
+Wide lines of text are difficult to read and are proven to make it harder for
+people to focus, while short lines of text force the eye to travel back too often,
+breaking a reader's rhythm. The `TemplateMain` component has a max width of 1280px,
+which is too wide for text. If your content includes long chunks of text, you
+should use the `TemplateMainNarrow` component instead.
```

Comment:
We need to mention that this type of scenario is only applicable when the sidebar is not used.